File: Latihan B/JAVA/Pc.java
public class Pc {
	// Atribut
	private Processor pr;
	private Disk dk;
	private Ram rm;
	private int totalPrice;

	// Konstruktor
	public Pc(Processor pr, Disk dk, Ram rm) {
		this.pr = pr;
		this.dk = dk;
		this.rm = rm;
		this.totalPrice = pr.getPrice() + dk.getPrice() + rm.getPrice();
	}

	// set untuk semua variable
	
	public void setProc(Processor pr) {
		this.pr = pr;
	}
	public void setDisk(Disk dk) {
		this.dk = dk;
	}
	public void setRam(Ram rm) {
		this.rm = rm;
	}
	public void setTPrice(Processor pr, Disk dk, Ram rm) {
		this.totalPrice = pr.getPrice() + dk.getPrice() + rm.getPrice();
	}

    // get untuk semua variable

	public Processor getProc() {
		return pr;
	}
	public Disk getDisk() {
		return dk;
	}
	public Ram getRam() {
		return rm;
	}
	public int getTPrice() {
		return totalPrice;
	}

	// tampil untuk semua item pada pc
	public void printPC() {
        System.out.println(" +PROCESSOR");
		pr.printProc();
        System.out.println(" +DISK");
		dk.printDisk();
        System.out.println(" +RAM");
		rm.printRam();

	}
}

File: Latihan A/C++/Pc.cpp

class Pc{
    private:
        int Totalprice;
        Processor pr;
        Disk dk;
        Ram rm;
    public:

        Pc(){

        }
        Pc(Processor p, Disk d, Ram r, int Totalprice){
            this->pr = p;
            this->rm = r;
            this->dk = d;
            this->Totalprice = Totalprice;
           
        }
        //set semua variaber
        void setTPrice(Processor p, Disk d, Ram r){
            this->Totalprice = p.getPricep() + d.getPriced() + r.getPricer();
        }

        void setProc(Processor p){
            this->pr = p;
        }

        void setDisk(Disk d){
            this->dk = d;
        }

        void setRam(Ram r){
            this->rm = r;
        }


        //get semua variabel
        int getTprice(){
            return this->Totalprice;
        }

        Processor getProc(){
           return this->pr;
        }

        Disk getDisk(){
            return this->dk;
        }

        Ram getRam(){
            return this->rm;
        }


        //tampilan untuk mengeluarkan semua item 
        void printPC(){
             cout << " +PROCESSOR" << endl;
             this->pr.printProc();
             cout << " +DISK" << endl;
             this->dk.printDisk();
             cout << " +RAM" << endl;
             this->rm.printRam();
        }
        
        ~Pc(){
            
        }
};

